airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -23,5 +23,10 @@ data:
   ab_internal:
     sl: 300
     ql: 400
+  releases:
+    breakingChanges:
+      4.0.0:
+        message: "Implementing event based incremental syncs - changing cursor fields."
```

Comment:
This message will be shown in the UI.  What is the user-facing benifit?  What action do users need to take to proceed?